bin/update-copyright-year

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+declare -r creator="${1-Sanctuary}"
+declare -r license="${2-LICENSE}"
+declare -r updated=LICENSE.updated
+declare -i count=0
+
+substitute() {
+  local -r year="$(git log --date=short --pretty=format:%ad | sort -r | head -n 1 | cut -d - -f 1)"
+  {
+    while IFS= read -r line ; do
+      local pattern='^Copyright [(]c[)] [0-9]{4} (.*)$'
+      if [[ "$line" =~ $pattern ]] && [[ "${BASH_REMATCH[1]}" == "$creator" ]] ; then
+        ((count+=1))
+        echo "Copyright (c) $year $creator"
+      else
+        echo "$line"
+      fi
+    done
+  }
+}
+substitute <"$license" >"$updated"
+
+if (( count == 0 )) ; then
+  rm "$updated"
+  echo "Copyright attribution not found for $creator" >&2
+  exit 1
+fi
+
+mv "$updated" "$license"
```

Comment:
Shouldn't this be forced? Won't bash complain that the file is already at this destination?